### PR TITLE
Homework 3 - Sorting algorithms/ bubble, insertion and selection sort

### PR DESCRIPTION
**Pull request by HeghineKhachatryan**
Homework 3 - Sorting algorithms/ bubble, insertion and selection sort/

### CODE REVIEW

File: src/main/java/arrays/homework_3/SortingAlgorithms.java [hunk 1]
```diff
@@ -1,10 +0,0 @@
-package arrays.homework_3;
-
```
Comment (mkrtichstepanyan) [inline src/main/java/arrays/homework_3/SortingAlgorithms.java:2]: Restore this file . You were not allowed to delete it

File: src/main/java/arrays/homework_3/homework_desctiption.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ Read From Java 8 book till the 5-th chapter
 EPAM Learn
 
 
-Write Algorithms in the ./arrays.homework_3.SortingAlgorithms.java file
+Write Algorithms in the ./arrays.homework_3.heghine_khachatryan.SortingAlgorithms.java file
```
Comment (mkrtichstepanyan) [inline src/main/java/arrays/homework_3/homework_desctiption.txt:7]: revert this change

File: src/main/java/arrays/homework_3/heghine_khachatryan/SortingAlgorithms.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            System.out.print(element + " | ");
+        }
+        System.out.println();
+    }
+
+    private void swap(int[] array, int i, int j) {
+        array[i] = array[i] ^ array[j];
+        array[j] = array[i] ^ array[j];
+        array[i] = array[i] ^ array[j];
+    }
```
Comment (mkrtichstepanyan) [inline src/main/java/arrays/homework_3/heghine_khachatryan/SortingAlgorithms.java:62]: Perfect solution . Good

**Review by mkrtichstepanyan — CHANGES_REQUESTED** [review]

File: src/main/java/arrays/homework_3/homework_desctiption.txt (hunk 2 above)
Comment (HeghineKhachatryan) [reply src/main/java/arrays/homework_3/homework_desctiption.txt:7]: done

File: src/main/java/arrays/homework_3/SortingAlgorithms.java (hunk 1 above)
Comment (HeghineKhachatryan) [reply src/main/java/arrays/homework_3/SortingAlgorithms.java:2]: done

**Review by mkrtichstepanyan — APPROVED** [review]
